src/celengine/solarsys.cpp

```diff
@@ -518,7 +509,7 @@ static bool CreateTimeline(Body* body,
         const Timeline* timeline = body->getTimeline();
         if (timeline->phaseCount() == 1)
         {
-            const TimelinePhase* phase = timeline->getPhase(0);
+            auto phase = timeline->getPhase(0);
```

Comment:
here you can use raw pointers to speedup a little bit:
`auto phase = timeline->getPhase(0).get();`